rpm/README.md

```diff
@@ -0,0 +1,99 @@
+# HOW-TO Maintain RPM lists for containers when adding new target platforms and architectures
+
+## Overview
+
+The file `rpms/BUILD.bazel` provides a list pinned rpms to be built in each category of CDI containers
+
+These are maintained with bazeldnf with rpm repos and names of packages  specified in the following three files:
+
+ 1. `.bazelrc`
+ 2. `repo.yaml`
+ 3. `hack/build/rpm-deps.sh` 
+
+Then, running `make rpm-deps` invokes bazeldnf installed in the bazel cdi builder container volume to populate the `rpm/BUILD.bazel` file 
+
+## Prerequisites and Caveats
+
+`bazeldnf` is not provided in the persistent CDI build container; it is built and installed from a pinned source tarball in the `make rpm-defs` step as per https://github.com/kubevirt/containerized-data-importer/blob/main/WORKSPACE#L83-L95 -- so if you _exec_ into the running container, you won't find the executable of bazeldnf in your bash path; rather it is only made available to bazel build as a package, e.g.:
+
+```
+[cfillekes@m1325001 containerized-data-importer]$ podman ps
+CONTAINER ID  IMAGE                                                    COMMAND               CREATED        STATUS        PORTS       NAMES
+3c4b6fed495d  icr.io/kubevirt/kubevirt-cdi-bazel-builder:native-s390x  hack/build/bazel-...  4 minutes ago  Up 4 minutes              kubevirt-cdi-volume-bazel-server
+[cfillekes@m1325001 containerized-data-importer]$ podman exec -it 3c4b6fed495d /bin/bash
```

Comment:
As an alternative to these two podman commands you can run `./hack/build/bazel-docker.sh bash`.  It will start the builder container if it is not running.  Otherwise it will exec into the container

Reply:
I will try that and add it to the README, then.  Thanks, Michael!